pkg/controller/pxc/config.go

```diff
@@ -0,0 +1,301 @@
+package pxc
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	api "github.com/percona/percona-xtradb-cluster-operator/pkg/apis/pxc/v1"
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/k8s"
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/pxc/app/config"
+)
+
+func (r *ReconcilePerconaXtraDBCluster) reconcileConfigMaps(ctx context.Context, cr *api.PerconaXtraDBCluster) (controllerutil.OperationResult, error) {
+	result := controllerutil.OperationResultNone
+
+	res, err := r.reconcileAutotuneConfigMap(ctx, cr)
+	if err != nil {
+		return result, errors.Wrap(err, "reconcile autotune config")
+	}
+	if result == controllerutil.OperationResultNone {
+		result = res
+	}
```

Comment:
given L22, `result == controllerutil.OperationResultNone` is always true. I believe we can simply assign the `res` to `result` directly here.